docker/Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+# To build this container, go to ESMValCore root folder and execute :
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
```

Comment:
Maybe it would be nice to use the name of the tag on dockerhub here, e.g. `-t esmvaltool:latest` for this file? Or would that be confusing?